draft-ferrieuxhamchaoui-quic-lossbits.md

```diff
@@ -285,8 +285,9 @@ The use of the loss bits is negotiated using a transport parameter:
 
 loss_bits (0x1055):
 
-: The loss bits transport parameter is an integer value that can be set
-  to 0 or 1 indicating the level of QL bits support.
+: Presence of the loss bits transport parameter indicates that the sender
+  supports the loss bits extension.  This parameter is an integer value that
+  can be set to 0 or 1 indicating the level of QL bits support.

```

Comment:
I think this is either redundant (clearly the implementation is following this draft by sending this TP) or vague (what exactly does "support" of the extension entail?).  I think PR #5 is a bit more direct about this.